src/main/java/com/denizenscript/clientizen/events/ScreenOpenCloseEvent.java

```diff
@@ -15,6 +15,23 @@
 
 public class ScreenOpenCloseEvent extends ScriptEvent {
```

Comment:
possible small language entry on what exactly a screen is?

Reply:
What could that have? just something like `"Screen" refers to menus or interfaces, such as: inventories, books...`?